src/stores/VideoChannelStore.ts

```diff
@@ -193,6 +194,7 @@ export default class VideoChannelStore extends AsyncStoreWithClient<null> {
 
         this.connected = true;
         messaging.once(`action:${ElementWidgetActions.HangupCall}`, this.onHangup);
+        this.matrixClient.on(RoomEvent.MyMembership, this.onMyMembership);
```

Comment:
This should either go in `onReady` or sub directly to the room emitter IMO

e.g `this.matrixClient.getRoom(roomId).on(RoomEvent.MyMembership, this.onMyMembership);` and then get rid of the roomId check in onMyMembership?